src/Discovery/Discovery.tsx

```diff
@@ -157,7 +157,7 @@ export const renderFieldContent = (content: any, contentType: 'string'|'paragrap
 };
 
 const highlightSearchTerm = (value: string, searchTerm: string, highlighClassName = 'matched'): {highlighted: React.ReactNode, matchIndex: number} => {
-  const matchIndex = value.toLowerCase().indexOf(searchTerm.toLowerCase());
+  const matchIndex = value ? value.toLowerCase().indexOf(searchTerm.toLowerCase()) : -1;
```

Comment:
Something in my data or config was triggering an error here. I haven't been able to reproduce the error after loading more data and doing another aggMDS.

Reply:
I think this will happen if a searchbale field doesn't have any value in metadata, which makes sense to have this fix